install-swagger-ui.sh

```diff
@@ -0,0 +1,8 @@
+# Copyright 2015, EMC, Inc.
+
+cd ./static
+rm -rf swagger-ui
+curl -L -o swagger-ui.zip https://github.com/zyoung51/swagger-ui/archive/master.zip
```

Comment:
Should this be applied for web-ui stuff that @rolandpoulter committed?  I believe that references a personal fork.